symfony/messenger/4.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "MESSENGER_ADAPTER_DSN": "amqp://guest:guest@localhost:5672/%2f/messages"
+    },
+    "aliases": ["messenger", "amqp"]
```

Comment:
I'd also add "messaging" alias, WDYT?

Reply:
What about `command_bus` too? Even if that's a bit reductive considering the wider component scope, that might be a good way to discover it.

Reply:
I like it: when making presentations about flex & hexagonal architecture, it's always badass to say "We need a command bus, so let's do `composer req command_bus` 😄

Reply:
I'm not sure about "messaging" tbh. `command-bus` might be better as a meta-package (or other package), which will use the Messenger and auto-enable specific middlewares for command buses (see [here](https://github.com/symfony/symfony/issues/26652#issuecomment-375893089))